### PR TITLE
chore: Prepare changelog for 0.9.1

### PR DESCRIPTION
**Pull request by spacegospod**
**Summary of Pull Request**

I'm just adding an entry to the changelog for 0.9.1

**Type of Pull Request**

- [ ] This is a bug fix.
- [ ] This is an enhancement or feature.
- [ ] This is a code style/formatting update.
- [ ] This is a documentation update.
- [ ] This is a refactoring update.
- [X] This is a chore update
- [ ] This is something else.
      Please describe:

**Related to Existing Issues**

Issue Number: N/A

**Test and Documentation Coverage**

For bug fixes or features:

- [ ] Tests have been completed.
- [ ] Documentation has been added/updated.

**Breaking Changes?**

- [ ] Yes, there are breaking changes.
- [X] No, there are no breaking changes.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,14 @@
 # CHANGELOG
 
+## [v0.9.1](https://github.com/vmware/terraform-provider-vcf/releases/tag/v0.9.1)
+
+> Release Date: June 12 2024
+
+BUG FIXES:
+* Make Tier 0 and Tier 1 routers are now optional for Edge Clusters [\#177](https://github.com/vmware/terraform-provider-vcf/issues/177)
```
Comment (dimitarproynov) [inline CHANGELOG.md:8]: You can remove the "make"

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,14 @@
...
+* Accept VLAN 0 for network pools [\#175](https://github.com/vmware/terraform-provider-vcf/issues/175)
```
Comment (dimitarproynov) [inline CHANGELOG.md:9]: Maybe add quotes around the zero?

**Review by dimitarproynov — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (spacegospod) [reply CHANGELOG.md]: Thanks

File: CHANGELOG.md (hunk 2 above)
Comment (spacegospod) [reply CHANGELOG.md]: Okay, sure

**Review by dimitarproynov — APPROVED** [review]

**Pull request merged**